misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/AnyAssertionsSpec.kt

```diff
@@ -746,6 +756,50 @@ abstract class AnyAssertionsSpec(
         }
     }
 
+    prefixedDescribe("because") {
+        val becauseFun = because.lambda
+        val becauseFunForInt = becauseInt.lambda
+
+        fun Expect<String>.containsBecause(reason: String) =
+            contains.exactly(1).regex("$separator\\Q${informationBulletPoint}${BECAUSE.getDefault().format(reason)}\\E")
```

Comment:
Why is this a regex in the first place? is `$separator` a regex?

Reply:
Actually, does not need to be one in this case as we don't deal with multi line with stuff inbetween as in other places. Already though about simplifying it myself but then did not bother. Going to improve it :+1: